tests/scripts/thread-cert/border_router/test_multi_ail.py

```diff
@@ -74,47 +79,59 @@ class TwoBorderRoutersOnTwoInfrastructures(thread_cert.TestCase):
     def test(self):
         br1: OtbrNode = self.nodes[self.BR1]
         br2: OtbrNode = self.nodes[self.BR2]
+        br3: OtbrNode = self.nodes[self.BR3]
 
         # start nodes
-        br1.start()
-        self.simulator.go(2)
-        br2.start()
+        for br in [br1, br2, br3]:
+            br.start()
         self.simulator.go(config.BORDER_ROUTER_STARTUP_DELAY)
 
-        # check roles
-        self.assertEqual('leader', br1.get_state())
-        self.assertEqual('router', br2.get_state())
-
-        # check two BRs AIL are in different subnets
+        # check BR1 and BR2 are not on the same subnet, but BR2 and BR3 are on the same subnet
         br1_infra_ip_addr = br1.bash(IPV4_CIDR_ADDR_CMD)
         br2_infra_ip_addr = br2.bash(IPV4_CIDR_ADDR_CMD)
+        br3_infra_ip_addr = br3.bash(IPV4_CIDR_ADDR_CMD)
 
         self.assertEqual(len(br1_infra_ip_addr), 1)
         self.assertEqual(len(br2_infra_ip_addr), 1)
+        self.assertEqual(len(br3_infra_ip_addr), 1)
```

Comment:
from my understanding, `self.assertXXX` is to whether the functionality under test is acting as expected. However, it seems we are checking whether the code itself is running as expected, so perhaps we should use `assert`.
```suggestion
        assert len(br3_infra_ip_addr) == 1
```

With a lot of `self.assertXXXX`, it not easy to tell what behavior the test is verifying.

Reply:
Thanks for the suggestion, it helps. I will update